src/main/scala/com/databricks/labs/overwatch/MultiWorkspaceDeployment.scala

```diff
@@ -507,4 +507,19 @@ class MultiWorkspaceDeployment extends SparkSessionWrapper {
     println(msg)
   }
 
+  /**
+   * Returns the Overwatch parameters from config.
+   * @param workspaceId
+   * @return
+   */
+  def getParams(workspaceId: String = "") = {
```

Comment:
public funcs should always explicitly state return type Array[MultiWorkspaceParams]

Reply:
done